src/pages/patientView/PatientViewPage.tsx

```diff
@@ -47,6 +51,7 @@ interface IPatientViewState {
     cnaSegmentData: any;
     mutationData: any;
     hotspotsData?: IHotspotData;
+    cosmicData?: ICosmicData;
```

Comment:
@alisman @adamabeshouse @inodb technically all the type/interface definitions we have in general, except props and state, can be considered as our model and moved out of the view/component classes into separate model classes/files. Currently we don't really have a separation of model from view, maybe except the auto-generated API classes.